OWNERS

```diff
@@ -1,20 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
Can we leave this file in place? ... alternatively update it if there are users that are no longer involved?

Reply:
as we don't use prow anymore, this file doesn't do anything thats why I've replaced it with `CODEOWNERS` which uses the github configured teams.

Reply:
Understood, but we also look at this file in a manual way, so by removing it we loose some of the valuable info. Is there a way to add a "default" to the `CODEOWNERS` file to keep the rest of the users from this file?

Reply:
Cool - missed the "default" / * at the top of the `CODEOWNERS` file. There's a discrepancy between that group and the old file, but we can work on that separately.